aspnetcore/fundamentals/use-http-context.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Use HttpContext in ASP.NET Core
+author: jamesnk
+description: How to use HttpContext in ASP.NET Core.
+monikerRange: '>= aspnetcore-3.1'
+ms.author: jamesnk
+ms.date: 01/31/2022
+uid: fundamentals/use-httpcontext
+---
+# Use HttpContext in ASP.NET Core
+
+<xref:Microsoft.AspNetCore.Http.HttpContext> encapsulates all information about an individual HTTP request and response. An `HttpContext` instance is initialized when ASP.NET Core receives an HTTP request and is accessible by middleware and app frameworks such as Web API controllers, Razor Pages, SignalR, gRPC, and more.
```

Comment:
```suggestion
<xref:Microsoft.AspNetCore.Http.HttpContext> encapsulates all information about an individual HTTP request and response. An `HttpContext` instance is initialized when an HTTP request is received. The `HttpContext` instance is accessible by middleware and app frameworks such as Web API controllers, Razor Pages, SignalR, gRPC, and more.
```
Split up long sentence so it can be MTd (Machine Translated). They should know the context so dropping `when ASP.NET Core receives`